test/api/api_test.go

```diff
@@ -171,7 +169,11 @@ func RunApiServer(t *testing.T, binary string, storageApiHost string) string {
 
 	var stdout, stderr bytes.Buffer
 	apiUrl := fmt.Sprintf("http://localhost:%d", port)
-	cmd := exec.Command(binary, fmt.Sprintf("--http-port=%d", port))
+	args := []string{fmt.Sprintf("--http-port=%d", port)}
+	if repoPath != "" {
+		args = append(args, fmt.Sprintf("--repository-path=%s", repoPath))
+	}
+	cmd := exec.Command(binary, args...)
```

Comment:
@webrouse nechce se mi nastartovat server když mu zadám cestu k repozitáři. Když ho s tou samou cestou nastartuju manuálně, tak to jde hned. Nenapadá tě jak to zdebugovat? V stdout ani stderr toho `exec.Command` nic není.

![CleanShot 2022-05-19 at 14 30 32](https://user-images.githubusercontent.com/215660/169293711-18fc620b-370f-47c0-bc06-cbc99e0aee61.jpg)


![CleanShot 2022-05-19 at 14 25 53](https://user-images.githubusercontent.com/215660/169293208-30570c45-f5e8-476e-b5f8-ff1a139d0af2.jpg)

Reply:
Potřeboval jsem to jen sepsat, no. 😅